common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/text/DaxTextView.kt

```diff
@@ -133,6 +134,7 @@ constructor(
                     Button -> R.style.Typography_DuckDuckGo_Button
                     Caption -> R.style.Typography_DuckDuckGo_Caption
                     CaptionAllCaps -> R.style.Typography_DuckDuckGo_Caption_AllCaps
+                    Body1Mono -> R.style.Typography_DuckDuckGo_Body1_Mono
```

Comment:
can you move this under the other `Body1`? We don’t really need to keep an order since we are doing the mapping ourselves.